src/Payum/Core/Bridge/Guzzle/ClientTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Payum\Core\Bridge\Guzzle;
+
+use GuzzleHttp\ClientInterface as GuzzleClientInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\RequestInterface;
+use Payum\Core\Bridge\Psr\Http\ClientInterface;
+
+
+
+class ClientTest implements ClientInterface
+{
+    protected $client;
+
+    /**
+     * @param GuzzleClientInterface $client
+     */
+    public function  __construct(GuzzleClientInterface $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * @param RequestInterface $request
+     * @return ResponseInterface
```

Comment:
in general please follwo PSR-1\2 coding standards 

https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md